### ISSUE TITLE
Displacing components of SVG

### ISSUE THREAD

**Issue by ZaifSenpai** [dnfield/flutter_svg#641]
I am working on this package called [dice_bear](https://pub.dev/packages/dice_bear) and flutter_svg depends on it for generating widgets to show. Sometimes it does places some components (commonly the hair of avatars).

[Here](https://avatars.dicebear.com/api/adventurer/XSkZZETJ589HvPywlusgYaItCQ8UaVJl3o2EBxFJCfA81R2yjrwtpbq1FpGoH2LKTd9RBsdF0oVdoZlmR3seA7euMFVw0Qm9hXmBq9eXr.svg?size=150) is an example URL

```
String url = "https://avatars.dicebear.com/api/adventurer/XSkZZETJ589HvPywlusgYaItCQ8UaVJl3o2EBxFJCfA81R2yjrwtpbq1FpGoH2LKTd9RBsdF0oVdoZlmR3seA7euMFVw0Qm9hXmBq9eXr.svg?size=150";
return SvgPicture.network(url);
```
 should display the following image:
![Avatar](https://avatars.dicebear.com/api/adventurer/XSkZZETJ589HvPywlusgYaItCQ8UaVJl3o2EBxFJCfA81R2yjrwtpbq1FpGoH2LKTd9RBsdF0oVdoZlmR3seA7euMFVw0Qm9hXmBq9eXr.svg?size=150)

but it shows the image in widget like this:
<img width="148" alt="image" src="https://user-images.githubusercontent.com/16265942/148099882-a8c19b95-4285-404e-92c4-c998f8cc3849.png">

**Comment by Ionaru** [comment]
I think the issue is with the self-closing "`<g transform="translate(-161 -83)"/>`" in the SVG. This might be interpreted incorrectly.

**Comment by dnfield** [comment]
Fixed in 24e06f6

**Comment by dnfield** [comment]
Ugh. Fix is actually in c728d60 because I forgot to save things when making sure the test worked :)
